src/main/include/commands/SetHickeyPos.h

```diff
@@ -0,0 +1,24 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <frc2/command/CommandHelper.h>
+#include <frc2/command/InstantCommand.h>
+#include "subsystems/DooHickey.h"
+
+class SetHickeyPos
+    : public frc2::CommandHelper<frc2::InstantCommand,
+                                 SetHickeyPos> {
+ public:
+  SetHickeyPos(DooHickey* dooHickey, double position);
```

Comment:
Use a reference for `dooHickey`

```suggestion
  SetHickeyPos(DooHickey& dooHickey, double position);
```